extensions/sql-database-projects/src/dialogs/publishOptionsDialog.ts

```diff
@@ -73,20 +73,23 @@ export class PublishOptionsDialog {
 			await this.updateOptionsTable();
 
 			// Get the description of the selected option
+			// selectedRows[0] contains selected row number
+			// data[row][1] contains the option display name
 			this.disposableListeners.push(this.optionsTable.onRowSelected(async () => {
 				const row = this.optionsTable?.selectedRows![0];
-				const label = this.optionsModel.optionsLabels[row!];
+				const label = this.optionsTable?.data[row!][1];
 				await this.descriptionText?.updateProperties({
-					value: this.optionsModel.getDescription(label)
+					value: this.optionsModel.getOptionDescription(label)
 				});
 			}));
 
 			// Update deploy options value on checkbox onchange
+			// data[row][1] contains the option display name
 			this.disposableListeners.push(this.optionsTable.onCellAction!((rowState) => {
 				const checkboxState = <azdataType.ICheckboxCellActionEventArgs>rowState;
 				if (checkboxState && checkboxState.row !== undefined) {
-					const label = this.optionsModel.optionsLabels[checkboxState.row];
-					this.optionsModel.optionsLookup[label] = checkboxState.checked;
+					let label = this.optionsTable?.data[checkboxState.row][1];
```

Comment:
nit: const doesn't need to be changed to let

Reply:
Oh yeah, noticed this one and modidied already in SC related PR, ill update here too!